doc/manual/writing_a_driver.rst

```diff
@@ -106,6 +113,34 @@ The controller's code would contain something similar to this: ::
         args = _get_args()
         simple_server_loop(Hello(), args.bind, args.port)
 
+The driver's server should be saved in ~/artiq-dev/artiq/artiq/devices/driver_name/server.py.
+
+__init__.py
+-----------
+This file must exist for porper integration with ARTIQ. It can be empty.
+~/artiq-dev/artiq/artiq/devices/driver_name/__init__.py
+
+Device Manager Udev Rules
```

Comment:
For the serial ports `adduser user dialout` and `serial.serial_for_url()` is much simpler, more powerful, and also works under windows.